README.MD

```diff
@@ -41,11 +41,19 @@ npm config set no-proxy localhost,127.0.0.1,example.org
 Additionally proxy settings found in the environment variables `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` will be used if they are not defined in the `.npmrc` file. 
 
 ### Install a concrete revision
-If you want to specify the revision of Chromium to be installed, just set the environment variable CHROMIUM_REVISION to the number of the revision you want to install, as in:
+If you want to specify the revision of Chromium to be installed, just set the environment variable `CHROMIUM_REVISION` to the number of the revision you want to install, as in:
 ```shell script
 export CHROMIUM_REVISION=729994
 ```
 
+### Use a Download Mirror
+You may download a specific revision from an alternate download host using the environment variable `CHROMIUM_DOWNLOAD_HOST`, for example:
+
+```bash
+export CHROMIUM_REVISION=737027
+export CHROMIUM_DOWNLOAD_HOST=https://npm.taobao.org/mirrors/chromium-browser-snapshots/
+```
+
```

Comment:
If we let people specify mirror URL, we need to specify the structure for which it will work (in case they want to use theirs own mirror).
We need to specify in docs that specific binary should be accessible similar to original CDN using URL: `<CHROMIUM_DOWNLOAD_HOST>/<PLATFORM_ARCHITECTURE>/<REVISION>/<OS_CHROMIUM_FILE_NAME>.zip?alt=media`
Where:
```
PLATFORM_ARCHITECTURE - ...
OS_CHROMIUM_FILE_NAME - ...
```

Reply:
I've added some documentation and a link to an example mirror which people can use to see how their own mirrors should be structured.